src/pages/index.js

```diff
@@ -1,92 +1,177 @@
-  const {siteConfig = {}} = context;
+  const { siteConfig = {} } = context;
   return (
     <Layout
       // title for OG
-      title={'Modern Fullstack App Platform'}
-      description="KintoHub is an all-in-one platform to combine and deploy your backend services, websites, cron jobs, databases and everything your app needs in one place.">
-      <header className={classnames('hero hero--primary', styles.heroBanner)}>
-        <div className="container">
-          <h1 className="hero__title">{siteConfig.title}</h1>
-          <p className="hero__subtitle">{siteConfig.tagline}</p>
-          <div className={styles.buttons}>
-            <Link
-              className={classnames(
-                'button button--outline button--secondary button--lg',
-                styles.getStarted,
-              )}
-              to={useBaseUrl('getting-started/introduction')}>
-              Get Started
-            </Link>
-          </div>
+      title={"Modern Fullstack App Platform"}
+      description="KintoHub is an all-in-one platform to combine and deploy your backend services, websites, cron jobs, databases and everything your app needs in one place."
+    >
+      <header className={styles.heroBanner}>
+        <img className={styles.heroBannerLeft} src="/img/docs-left.svg"></img>
+        <img className={styles.heroBannerRight} src="/img/docs-right.svg"></img>
+        <div className={styles.heroBannerText}>
+          <h1>{siteConfig.title}</h1>
+          <span>{siteConfig.tagline}</span>
         </div>
       </header>
       <main>
-        {features && features.length > 0 && (
-          <section className={styles.features}>
-            <div className="container">
-              <div className="row">
-                {features.map((props, idx) => (
-                  <Feature key={idx} {...props} />
-                ))}
+        <section className={styles.infoCardSection}>
+          <div className={styles.infoCardsIntro}>
+            <Link
+              style={{ color: "inherit", textDecoration: "inherit" }}
+              to={useBaseUrl("getting-started/introduction")}
+              className={styles.infoCardSmall}
+            >
+              <div>
+                <img className={styles.icon} src="/img/playlist-plus.svg"></img>
+                <h3>Introduction</h3>
+                <p>Why should you use KintoHub?</p>
+              </div>
+            </Link>
+            <Link
+              style={{ color: "inherit", textDecoration: "inherit" }}
+              to={useBaseUrl("getting-started/fullstack-example")}
+              className={styles.infoCardSmall}
+            >
+              <div>
+                <img className={styles.icon} src="/img/code-tags.svg"></img>
+                <h3>Try a Full Stack Example</h3>
+                <p>Deploy PostgreSQL + Hasura + React.</p>
               </div>
+            </Link>
+            <Link
+              style={{ color: "inherit", textDecoration: "inherit" }}
```

Comment:
comment: avoid using inline styles if we are using `className` already

Reply:
For some reason router Link styles were not getting overriden with css classes so had to use in line. Will check again